src/main.ts

```diff
@@ -15,6 +16,21 @@ export async function run() {
     let version = core.getInput('node-version');
     if (!version) {
       version = core.getInput('version');
+
+      if (!version) {
+        const versionFile = core.getInput('node-version-file');
+
+        if (!!versionFile) {
+          const versionFilePath = path.join(
+            process.env.GITHUB_WORKSPACE!,
+            versionFile
+          );
+          version = installer.parseNodeVersionFile(
```

Comment:
1) We need to add validation to check if specified file exists and throw exception if it doesn't
2) We need to throw warning / error if both `version` and `node-version-file` are specified since it looks like incorrect using of action.

Reply:
@hkaur008 @Xlient please add validation to check if specified file exists as @maxim-lobanov mentioned.

As for the second point, I would prefer to add a warning. Could you please add it?

Reply:
done